.gitignore

```diff
@@ -5,3 +5,7 @@ target
 .settings
 .clover
 .*
+.idea
```

Comment:
I'm going to merge without `.idea`. If it's necessary, let me know.

Sorry for the huge delay.